exp/addons/controllers/suite_test.go

```diff
@@ -46,6 +46,11 @@ func TestMain(m *testing.M) {
 		panic(fmt.Sprintf("undable to setup machine node index: %v", err))
 	}
 
+	// Set up the MachineProviderIDIndex
+	if err := noderefutil.AddMachineProviderIDIndex(ctx, env.Manager); err != nil {
+		panic(fmt.Sprintf("undable to setup machine providerID index: %v", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("unable to setup machine providerID index: %v", err))
```
same in the original(s) (can you maybe to a search and replace? :))